.travis.yml

```diff
@@ -3,9 +3,6 @@ language: go
 services:
   - docker
 
-install:
```

Comment:
once this gets merged in the main repo i'll replace travis with github-actions 😉